src/Visualization/VisualizationData.jl

```diff
@@ -0,0 +1,25 @@
+"""
+"""
+function visualization_data(trian::Triangulation; order=-1, nsubcells=-1, celldata=Dict(), cellfields=Dict())
```

Comment:
Should we mark this function experimental? I think we might want to change it in future. Like returning a `struct` instead of a named tuple etc.

Reply:
Don't worry about this.